site/app/_components/DocsEmbed.tsx

```diff
@@ -148,7 +168,7 @@ function reactCode(theme: string, mediaElement: any, mediaPackage: string) {
   ];
 }
 
-function vueCode(theme: string, mediaElement: any, mediaPackage: string) {
+function vueCode(theme: string, mediaElement: any, mediaPackage: string, customProperties: CustomProperties) {
```

Comment:
Vue's the only language I don't know for sure, but it worked in CodeSandbox, and Copilot told me it wasn't not idiomatic.